Update manage-policy-sets.mdx (#767)
hashicorp/terraform-docs-common: website/docs/cloud-docs/policy-enforcement/manage-policy-sets.mdx

You can set Sentinel parameters when you [edit a policy set](#edit-policy-sets).

> Parameters can only be set for policy sets added through a connected VCS repository or when policy sets are created with the [HCP Terraform Policy Sets API](/terraform/cloud-docs/api-docs/policy-sets) or the `tfe` provider [`tfe_policy_set`](https://registry.terraform.io/providers/hashicorp/tfe/latest/docs/resources/policy_set) resource and then a policy set version is uploaded with a file containing the policy rules. Individually managed policies created through the UI, API or TFE provider do not support parameter attachment.

Reviewer: ```suggestion
You can only set parameters for policy sets added through:
* A connected VCS repository
* The [policy sets API](/terraform/cloud-docs/api-docs/policy-sets)
* The `tfe` provider's [`tfe_policy_set`](https://registry.terraform.io/providers/hashicorp/tfe/latest/docs/resources/policy_set) resource

Individually managed policies created through HCP Terraform's UI, API, or the TFE provider do not support attaching parameters.
```
